_posts/2017-02-18-digest-authentication-with-ruby-on-rails.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title: "Digest: Authentication with Ruby on Rails"
+date: 2017-02-16 12:00
+author: Herbert Ilhan Tanujaya
+categories: [Digest]
+---
+
+## Introduction
+Nowadays, authentication systems are present in nearly all websites (with at
+least a username and password system in place). In this article, we will create
+a simple authentication system in the popular web application framework Ruby on
+Rails. This article is mainly aimed at those who are just getting started with
+your first web application (but with some knowledge of Rails).
+
+When I first learned Rails (and web development in general), I wanted to create
+a website for my organization. Since the website needs an authentication system,
+I searched for gems that help me. One of the most popular gems is
```

Comment:
One requirement for the website was to have an authentication system, so I searched for Ruby gems that could help with this.